.github/workflows/dashboard-test.yml

```diff
@@ -0,0 +1,21 @@
+name: Test dashboards
```

Comment:
If we add these to our Drone jobs instead, we can have it use the agent build image where `jb` is installed.

Reply:
Thanks! I got this to work on github actions as well with the agent build image, but it took longer. So I'll stick to drone for now.